src/sagemaker/image_uri_config/tensorflow.json

```diff
@@ -285,7 +285,8 @@
             "2.5": "2.5.1",
             "2.6": "2.6.3",
             "2.7": "2.7.0",
-            "2.8": "2.8.0"
+            "2.8": "2.8.0",
+            "2.9": "2.9.0"
```

Comment:
@tejaschumbalkar training does not have a `2.9.0` image but a `2.9.1` image [here](https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/image_uri_config/tensorflow.json#L1546). This causes the tests based on `tf_full_version` to fail. We have 2 options:

1. Can we switch to consistent TF Version for inference `2.9.1`?
2. Fix the failing tests, [current logic](https://github.com/aws/sagemaker-python-sdk/blob/ce7e4cdee655da386716ad48620b70d05a155064/tests/conftest.py#L440) is that it tries to get a minimum of latest training and inference versions and expects the versions to be consistent for training/inference.

Reply:
This PR was created when TF2.9.0 was released.